rest-core/src/Rest/Run.hs

```diff
@@ -2,6 +2,9 @@
 module Rest.Run
   ( apiToHandler
   , apiToHandler'
+  , apiToHandlerWithConfig'
```

Comment:
I'd rename this to `apiToHandlerWith`. You can drop the config, and definitely the quote.